shared/constants/favorite.js

```diff
@@ -6,4 +6,7 @@ import type {Props as FolderProps} from '../folders/render'
 export const favoriteList = 'favorite:favoriteList'
 export type FavoriteList = TypedAction<'favorite:favoriteList', {folders: FolderProps}, void>
 
-export type FavoriteAction = FavoriteList
+export const favoriteIgnore = 'favorite:favoriteIgnore'
+export type FavoriteIgnore = TypedAction<'favorite:favoriteIgnore', {error?: true}, void>
```

Comment:
This second type argument is the non error payload, the third is the error payload. I think these would both be void.